Client/src/components/common/Table/index.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import Cell from './cell';
+import './index.css';
+
+
+const Table = ({ headings, rows }) => {
+  return (
+    <table className="table">
+      <thead className='table-head'>
+        <tr>
+          {headings.map((head, index) =>
+            <Cell
+              key={index}
+              content={head}
+              header={true}
+            />)
+          }
+        </tr>
+      </thead>
+      <tbody>
+        {rows.map((row, index) => {
+          return (
+            <tr className='table-row' key={index}>
+              {row.map((rowcontent, index) => {
+                return (
+                  <Cell
+                    key={index}
+                    content={rowcontent}
+                  />
```

Comment:
@knanahassouna1  for the key what do think is it ok to give the key index
or give every row or any data id from the database ?????